backend/testfiles/execution/stdlib/json.dark

```diff
@@ -671,40 +987,137 @@ Json.serialize<Person2> (Person1 { Name = "Bob"; Age = 42 }) = Test.derrorMessag
 Expected: (arg: 'a)
 Actual: a Person1: Person1 {\n  Age: 42,\n  Name: \"Bob\"\n}"
 
-// type StringResult<’a> = PACKAGE.Darklang.Stdlib.Result.Result<’a, String>
```

Comment:
where these commented out for a reason? They were failing because we had `<’a>` instead of `<'a>` (`’` vs `'` )

Reply:
I don't know any reason, personally.